docs/product/releases/setup/release-automation/github-actions/index.mdx

```diff
@@ -1,14 +1,81 @@
 ---
 title: GitHub Actions
-description: "Learn how Sentry and GitHub Actions automate release management and help identify suspect commits."
+description: "Learn how Sentry and GitHub Actions automate release management, upload source maps and help identify suspect commits."
 ---
 
-<Alert level="warning">
+The [Sentry Release GitHub Action](https://github.com/marketplace/actions/sentry-release) automates Sentry release management in GitHub with just one step. After sending Sentry release information, you’ll be able to associate commits with releases. Additionally, releases are used for applying source maps to view original, un-minified source code.
 
-If you make changes to your organization slug, you'll need to update your configuration for this integration. Learn more in our [troubleshooting guide](/organization/integrations/troubleshooting).
+For more details about Sentry release management concepts, see the full documentation on [releases](/product/releases/).
 
-</Alert>
+## Prerequisites
 
-The [Sentry Release GitHub Action](https://github.com/marketplace/actions/sentry-release) automates Sentry release management in GitHub with just one step. After sending Sentry release information, you’ll be able to associate commits with releases. Additionally, releases are used for applying source maps to view original, untransformed source code.
+For the Sentry Release GitHub action to communicate securely with Sentry, you'll need to configure an [Organization Auth Token](https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/).
 
-For more details about Sentry release management concepts, see the full documentation on [releases](/product/releases/).
+Alternatively, you can also use a [User Auth Token](https://sentry.io/orgredirect/organizations/:orgslug/settings/account/api/auth-tokens/), with the "Project: Read & Write" and "Release: Admin" permissions.
```

Comment:
```suggestion
```
we don't really want to promote these anymore. People should use an org auth token, and just leave it at that